src/jit/rationalize.cpp

```diff
@@ -157,6 +157,13 @@ void Rationalizer::RewriteNodeAsCall(GenTree**             use,
     // Create the call node
     GenTreeCall* call = comp->gtNewCallNode(CT_USER_FUNC, callHnd, tree->gtType, args);
     call              = comp->fgMorphArgs(call);
+    // Determine if this call has changed any codegen requirements.
+    comp->fgCheckArgCnt();
+    if ((call->gtFlags & GTF_EXCEPT) != 0)
+    {
+        comp->codeGen->setFramePointerRequiredEH(true);
+    }
```

Comment:
I should have removed this. However FWIW, I believe that most calls don't have the GTF_EXCEPT flag - only if their args can throw an exception (e.g. indirections or bounds checks).

---
In reply to: [81881696](https://github.com/dotnet/coreclr/pull/7486#discussion_r81881696) [](ancestors = 81881696)